relay-server/src/endpoints/attachments.rs

```diff
@@ -58,12 +58,13 @@ fn store_attachment(
 }
 
 pub fn configure_app(app: ServiceApp) -> ServiceApp {
-    let url_pattern = r"/api/{project:\d+}/events/{event_id:[\w-]+}/attachments{t:/}";
-
     common::cors(app)
-        .resource(url_pattern, |r| {
-            r.name("store-attachment");
-            r.method(Method::POST).with(store_attachment);
-        })
+        .resource(
+            &common::normpath(r"/api/{project:\d+}/events/{event_id:[\w-]+}/attachments/"),
```

Comment:
_nit_: Can you re-introduce the variable here to return to the old code-folding?